src/figuresClasses.ts

```diff
@@ -1,19 +1,96 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  color: Color;
+  shape: Shape;
+  getArea: () => number;
 }
 
+const semiPerimeter = (a: number, b: number, c: number): number => {
+  return (a + b + c) / 2;
+};
```

Comment:
Move func to a static method of Triangle class because it's don't connects to the other figure